java-agent/java-agent.gradle

```diff
@@ -37,28 +37,27 @@ CopySpec isolateSpec(Collection<Task> sourceTasks) {
   }
 }
 
+//Includes everything needed for OOTB experience
 shadowJar {
-  archiveClassifier = 'no-exporters'
-
-  def sourceTasks = [project(':instrumentation').tasks.shadowJar]
+  def sourceTasks = [project(':instrumentation').tasks.shadowJar, project(':auto-exporters').tasks.shadowJar]
   dependsOn sourceTasks
   with isolateSpec(sourceTasks)
 }
 
-task embedExporters(type: ShadowJar) {
+//Includes instrumentations, but not exporters
+task lightShadow(type: ShadowJar) {
```

Comment:
maybe `defaultArtifactShadow`? i also like `lightShadow` though

Reply:
Honestly, I don’t think anybody will run this task manually. So does not really matter, what is its name, as long as it is somewhat descriptive. And we need to laugh sometimes :)